test.js

```diff
@@ -5,27 +5,202 @@ import tempy from 'tempy';
 import execa from 'execa';
 import fileIcon from '.';
 
+const processId = async app => {
+	const {stdout} = await execa('pgrep', [app]);
+	return parseInt(stdout.split('\n')[0], 10);
+};
+
+const tmpFile = () => tempy.file({extension: 'png'});
+
+test('argument validation', async t => {
+	const finderPid = await processId('Finder');
+
+	await t.notThrowsAsync(() => fileIcon.buffer('Safari'));
```

Comment:
```suggestion
	await t.notThrowsAsync(fileIcon.buffer('Safari'));
```

Same for the others.